frontend/src/metabase/common/hooks/use-collection-list-query/use-collection-list-query.unit.spec.tsx

```diff
@@ -25,6 +26,17 @@ const TestComponent = () => {
       {data.map(collection => (
         <div key={collection.id}>{collection.name}</div>
       ))}
+      <div data-testid="metadata">
```

Comment:
All of the tests reflect this pattern. Essentially, if we decide to add new metadata to one of these endpoints, then we should reflect those new pieces of data in the response as well as the test.

Reply:
Should we just render `No metadata` or something like this? The tests look misleading, starting from the name "should show metadata from the response"  where we actually check that no metadata is returned. I'd also remove `Object.entries` block since it's not supposed to be used, and leave it only in search results.

Reply:
@ranquild  Yeah that definitely makes sense - I got too focused on making a "one size fits all" solution for these tests. I can change the other `list` query tests to just say 'No Metadata' and explicitly state the fields that we're testing for in the `useSearchListQuery` tests